src/routes/docs/builders/tags-input/(sections)/Description.svelte

```diff
@@ -0,0 +1,351 @@
+<script lang="ts">
+	import { Docs } from '$routes/(components)';
+</script>
+
+<Docs.H2>Builder Description</Docs.H2>
+
+<Docs.P class="mt-4">
+	The following is an in-depth description of the return values from
+	<Docs.Code>createTagsInput()</Docs.Code>.
+</Docs.P>
+
+<Docs.H3><u>Stores</u></Docs.H3>
+<Docs.H4>options</Docs.H4>
+<Docs.P class="mt-4">
```

Comment:
We should avoid adding classes to the Docs components, to keep a consistent styling throughout the docs.

Reply:
Hunta does this same mt-4 with a lot of his P tags in the drop-down menu docs.

That is where I based this off of..

Reply:
Oh, I didn't notice that. Nonetheless, we should not be styling these. We can do certain rules instead, e.g. if a p follows an H3, then we add mt-4, something like that.

Pinging @huntabyte to hear his thoughts

Reply:
I’ll remove for now and yous can handle backend then

Reply:
I agree with you on this @TGlide - it was like your your first cigarette, once I started I couldn't stop! I'm going to clean this up in a separate PR!

Reply:
Sweet! I’ll update accordingly now your stuff is done